wine-tkg-git/wine-tkg-patches/proton/shared-gpu-resources/sharedgpures-fences.patch

```diff
@@ -131,7 +131,26 @@ index beb93d87ad2..01b2f048b39 100755
          if self.is_dispatchable():
              LOGGER.error("Unhandled native handle for: {0}".format(self.name))
          return None
-@@ -1893,7 +1926,7 @@ class VkParam(VkVariable):
+@@ -1480,7 +1513,8 @@ class VkMember(VkVariable):
+                 else:
+                     # Nothing needed this yet.
+                     LOGGER.warn("TODO: implement copying of static array for {0}.{1}".format(self.type, self.name))
+-            elif self.is_handle() and self.needs_unwrapping():
++            elif self.is_handle() and unwrap: ## Was self.need_unwrapping(). VkCommandBuffer command buffer wans't supposed to be converted to native handle from VkCommandBufferSubmitInfo conversion.
++                 #This was hitten in 32 bit conversion probably becouse it's a dispatchable type so conv == True. Force unwrap usage instead (unwrap is True for  Public Thunks, so direct call to uni side without PE implementation)
```

Comment:
Do we keep the typos for fun (they are funny yet don't make the sentence unreadable imo)?

Reply:
@Tk-Glitch fixed typos. (Kept hitten that while is considered archaic is correct)